ua-parser.js

```diff
@@ -47,8 +48,19 @@ const parseUA = (userAgent, browsers) => {
     data.browser.name += ' Android';
   }
 
+  data.fullVersion = ua.browser.version;
   // https://github.com/mdn/browser-compat-data/blob/master/docs/data-guidelines.md#safari-for-ios-versioning
-  data.fullVersion = data.browser.id === 'safari_ios' ? ua.os.version : ua.browser.version;
+  if (data.browser.id === 'safari_ios') {
+    data.fullVersion = ua.os.version;
+  }
```

Comment:
Can you go `else if` here to make it abundantly clear that `data.browser.id === 'safari_ios'` and  `ua.browser.name === 'Android Browser'` aren't expected to happen at the same time?

Reply:
Done!